community.hachyderm.io/content/en/docs/moderation/_index.md

```diff
@@ -0,0 +1,358 @@
+---
+title: "Moderation Guiding Principles"
+linkTitle: "Moderation Guiding Principles"
+weight: 4
+description: >
+  Hachyderm Moderation: Process and Responsibilities
+---
+
+This page includes:
+
+* Information about how we moderate Hachyderm.
+* The responsibilities of members (Hachydermians) and moderators
+(the "we/us" of this document).
+* Other supporting context relevant to the moderation process.
+
+We appreciate that this is probably not the page that you are
+coming to for light reading. Most likely you are here because you
+have been moderated and are trying to figure out what to do
+or you are a prospective user and you are trying to figure out if
+this instance is a safe home for you. We are here for you and have
+tried to build as comprehensive and clear a document as possible.
+
+For full transparency: we are currently designing our
+documentation architecture, so it is likely
+that this singular document will be split onto different pages for
+clarity. Please keep this in mind if you are bookmarking this
+page.
+
+(Skip to the [Help, I was moderated!](#help-i-was-moderated)
+section.)
+
+# Responsibilities 
+
+## Hachydermians
```

Comment:
this starts to feel more like a TOS than guidelines. it would be good to ensure we have that in place and don't contradict it here.

Reply:
Yeah it definitely started to exceed scope of "Guiding Principles". I was thinking of writing a blog post for the essentials and then breaking out the rest into separate docs.